cmd/clusterctl/cmd/root.go

```diff
@@ -118,6 +123,19 @@ func init() {
 	RootCmd.PersistentFlags().StringVar(&cfgFile, "config", "",
 		"Path to clusterctl configuration (default is `$HOME/.cluster-api/clusterctl.yaml`) or to a remote location (i.e. https://example.com/clusterctl.yaml)")
 
+	RootCmd.AddGroup(
```

Comment:
Q: what about creating a third group "Troubleshooting and Debugging Commands:" with describe (in future we will move also topology plan as soon as cluster class graduates to beta)

Reply:
I think it sounds like a good idea. What command would you like to see in this group?

Reply:
only describe for now

Reply:
Fixed